### PR TITLE
Apartment app rework

### PR DESCRIPTION
**Pull request by elysemontano**

### CODE REVIEW

File: apartment-app/apartment-app-intro.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+    - Props passed through Header to Navigation
+    - Link to Home (both for logged in and logged out user)
+    - Link to all apartments (both for logged in and logged out user)
+    - Link to my apartments when logged in
+    - Link to create an apartment when logged in
+    - Jest test for NavLink
+
+- Header/Footer
+    - Add navigation as needed
```
Comment (sjproctor) [inline apartment-app/apartment-app-intro.md:43]: What is different in this "this navigation as needed" that wasn't covered in the previous section?

File: apartment-app/apartment-app-intro.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+- Initial Setup (Located in apartment-app section in syllabus)
+    - Installs
+    - Devise
+    - React in Rails
+    - Reactstrap
+    - Add jest
+
+- Application Structure    
+    - Apartment Resource
+        - Associations (User has_many Apartments, Apartment belongs_to User)
```
Comment (sjproctor) [inline apartment-app/apartment-app-intro.md:24]: Should this include model validations?

File: apartment-app/apartment-app-intro.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+- Backend Index
+    - Rspec test for index method
+    - Controller method for index exists
+
+- Unprotected Index
+    - Refactor route to dynamically pass all apartments
+    - Render list of cards with each apartment displaying information for price, location, bath, bed, and image.
+    - Jest test for the index page
+    - Fetch call is created for index method to pull from database
+    - Basic styling
```
Comment (sjproctor) [inline apartment-app/apartment-app-intro.md:61]: Can the unprotected index include the backend index or is that too much?

**Review by sjproctor — COMMENTED** [review]
Couple thoughts. It looks great overall.

File: apartment-app/apartment-app-intro.md (hunk 1 above)
Comment (sjproctor) [reply apartment-app/apartment-app-intro.md]: Yeah combining them makes sense to me.

File: apartment-app/apartment-app-intro.md (hunk 3 above)
Comment (sjproctor) [reply apartment-app/apartment-app-intro.md]: Sounds good.

File: apartment-app/apartment-app-structure-setup.md [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+      <Footer />
+    </BrowserRouter>
+  )
+}
+
+export default App
+```
+
+### Styling
+For styling, I have found some people like to make separate stylesheets for each of these pages as well.  While this is fine, make sure that you are placing them in app/assets/stylesheets folder.
```
[the remark this answers is not in the document]
Comment (sjproctor) [reply apartment-app/apartment-app-structure-setup.md]: Maybe something like: "In Rails, all stylesheets must live in the app/assets/stylesheets directory. If your team chooses to create additional stylesheets they should live in the same directory within the asset pipeline and be imported directly to the application.scss file."

File: apartment-app/apartment-app-intro.md (hunk 2 above)
Comment (elysemontano) [reply apartment-app/apartment-app-intro.md:24]: Cool, I am adding it the request spec for 422 error onto the create and update cards and the model specs to apartment structure.

**Pull request merged**